server/mscalendar/views/calendar.go

```diff
@@ -11,6 +11,20 @@ import (
 	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
 )
 
+type Option interface {
+	Apply(remote.Event, *model.SlackAttachment)
+}
+
+type showTimezoneOption struct{}
+
+func (tzOpt showTimezoneOption) Apply(event remote.Event, attachment *model.SlackAttachment) {
+	attachment.Text = attachment.Text + " (" + event.Start.TimeZone + ")"
+}
+
+func ShowTimezoneOption() Option {
+	return showTimezoneOption{}
+}
```

Comment:
My idea here was to have a `views.RenderEventAsAttachment` that would receive different options based on where the event is going to be posted (DM vs Channel), since both would expose different things, but it can be solved as well by using a simple function to set the `attachment.Text`.